### ISSUE TITLE
Point tesseract-ocr and libtiff-ios submodules to most recent stable releases and update libs

### ISSUE THREAD

**Issue by kevincon** [gali8/Tesseract-OCR-iOS#214]

**Comment by ws233** [comment]
I've been trying to update Tesseract and have faced an issue. 
Neither of the tags from the upstream tesseract can be compiled. There is a broken commit `ec9581d8f2b3895e7307609043a3ec1bc9c603c5` right before the tag `3.04.00`, which doesn't allow to build Tesseract. And that commit has been reverted at `66a76a94772343582a70783a712f6248a81879ab`. So all the latest tags are between those commits and cannot be compiled.
So there are 2 ways available to fix this.
1. Wait for the next stable upstream Tesseract release marked with the next tag.
2. Update to any other commit before the broken one or after the reverted one. But there are no more tags. So it seems that those history points could be unstable.
So I suggest the first way and won't update the tesseract so far.

**Comment by kevincon** [comment]
I agree, I think 1 is the best option. Also I'm sorry I haven't reviewed your PRs yet; things are really busy at work but I'll try to take a look at them this weekend.

**Comment by podkovyrin** [comment]
There is new version of Tesseract `3.04.01` have released. I'm trying to compile it but stuck with following output from `make`
```
cd /Users/podkovyrin/Projects/Tesseract-OCR-iOS/TesseractOCR/libtiff-ios ; \
	until `/Applications/Xcode.app/Contents/Developer/usr/bin/make -s`; do sleep 5; done
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
curl: (78) RETR response: 550
/bin/sh: ../configure: No such file or directory
make[1]: *** [/Users/podkovyrin/Projects/Tesseract-OCR-iOS/TesseractOCR/libtiff-ios/libpng-1.6.20/arm-apple-darwin7/Makefile] Error 127
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
```

**Comment by ws233** [comment]
The wrong path to `libpng`
Ther is no files at this path anymore. You should manually fix the path to new files stored at libpng repo.

**Comment by podkovyrin** [comment]
@ws233 it worked, thank you

**Comment by ws233** [comment]
@podkovyrin, would you mind uploading a PR with your changes?

**Comment by podkovyrin** [comment]
Sure, as soon as corresponding PR will be merged https://github.com/ashtons/libtiff-ios/pull/13

**Comment by kkotar** [comment]
> ashtons merged commit 71ad425 into ashtons:master on Feb 25
